src/commands/database/mysql/user/create.tsx

```diff
@@ -0,0 +1,146 @@
+import { ExecRenderBaseCommand } from "../../../../lib/basecommands/ExecRenderBaseCommand.js";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../../rendering/process/process_flags.js";
+import { ReactNode } from "react";
+import { Flags, Args } from "@oclif/core";
+import { assertStatus } from "@mittwald/api-client-commons";
+import { Success } from "../../../../rendering/react/components/Success.js";
+import { Value } from "../../../../rendering/react/components/Value.js";
+
+type Result = {
+  mysqlUserId: string;
+};
+
+export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
+  static description = "Create a new mysql user";
+  static args = {
+    "database-id": Args.string({
+      description: "ID of the MySQL Database to create a user for",
+    }),
+  };
+  static flags = {
+    ...processFlags,
+    "database-id": Flags.string({
+      required: true,
+      description: "ID of the MySQL Database to create a user for",
+    }),
+    "access-level": Flags.string({
+      required: true,
+      description: "Access level for this mysql user",
```

Comment:
Inconsistent spelling of `MySQL` across all descriptions; I'd favor sticking with `MySQL` instead of `mysql` because that's the official product name.

Reply:
Will stick with MySQL for all mentions